CMakeLists.txt

```diff
@@ -1867,11 +1867,22 @@ if(LILV)
 endif()
 
 # Live Broadcasting (Shoutcast)
-find_package(Shout)
-cmake_dependent_option(BROADCAST "Live Broadcasting (Shoutcast) support" ON "Shout_FOUND" OFF)
+option(BROADCAST "Live Broadcasting (Shoutcast) support" ON)
 if(BROADCAST)
-  if(NOT Shout_FOUND)
-      message(FATAL_ERROR "Live Broadcasting support requires the libshout and its development headers.")
+  if(UNIX AND NOT APPLE)
+    # Check if system lib is at least 2.4.4 and not suffering bug
+    # https://bugs.launchpad.net/mixxx/+bug/1833225
+    find_package(Shout 2.4.4)
+    if(NOT Shout_FOUND OR Shout_VERSION VERSION_LESS 2.4.4)
+      # Fall back to internal libraray in the lib tree
+      message(STATUS "Preparing internal Shout")
```

Comment:
Maybe we should give a reason why internal libshout is used even though a system libshout is installed.